src/components/Calculator.js

```diff
@@ -0,0 +1,22 @@
+import Buttons from './button';
+import buttondata from './buttondata';
+
+const Calculator = () => {
```

Comment:
- [ Optional] Great job implementing the required changes from the previous reviewers, however, I noticed that your App styling is not consistent with the wireframe provided for this project. Please consider following the design guidelines provided for this project, following the design guidelines is an indication that as a developer you are able to bring UI designs to life.

## Your design
![Screenshot from 2023-02-23 00-20-34](https://user-images.githubusercontent.com/16392046/220763331-ba640c0c-f8bf-4c06-b84d-0ab6f818ed60.png)


## Project Wireframe
![](https://github.com/microverseinc/curriculum-react-redux/raw/main/math-magicians/images/calculator.png)